app/components/widgets/forms/image-upload.js

```diff
@@ -22,6 +23,12 @@ export default Component.extend({
     return this.needsCropper && !this.selectedImage.includes('http');
   }),
 
+  routing: Ember.inject.service('-routing'),
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use `import { inject } from '@ember/service';` instead of using Ember.inject.service](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=4869050)